### ISSUE TITLE
Suggested change in help message for "mandatory" and "optional" values

### ISSUE THREAD

**Issue by djerius** [polettix/App-Easer#4]
The message output for to indicate that an option's _value_ is  mandatory or optional

> optional string option
> mandatory string option
> optional integer, defaults to $default
> optional integer, current value incremented if omitted

is easily[1] confused with whether the option _itself_ is mandatory or optional.

Might I suggest rewording it as:

> string, value is optional
> string, value is mandatory
> integer, value is optional; defaults to $default
> integer, value is optional; current value incremented if omitted

[1] by my users, at least

**Comment by polettix** [comment]
This seems a pretty useful change. Please prod me if you don't see anything in a couple of weeks or so!

**Comment by polettix** [comment]
I just issued a TRIAL release with the suggested changes and the fix for #5 as well, let's see how it goes with CPAN Testers and then I'll close this and the other. Thanks for the suggestions.

**Comment by polettix** [comment]
I hope the changes in the latest release are fine now, please reopen if I didn't get the suggestions right.

**Comment by djerius** [comment]
Thanks!
